internal-links.tap

```diff
@@ -0,0 +1,25 @@
+TAP version 13
```

Comment:
I had added this to gitingore under #2870

Revert this file from the commit, update your branch